Src/FluentAssertions/Collections/GenericDictionaryAssertions.cs

```diff
@@ -512,10 +512,10 @@ public GenericDictionaryAssertions(IDictionary<TKey, TValue> dictionary) : base(
         /// <param name="becauseArgs">
         /// Zero or more objects to format using the placeholders in <see cref="because" />.
         /// </param>
-        public void BeEquivalentTo<TExpectation>(TExpectation expectation,
+        public AndConstraint<GenericDictionaryAssertions<TKey, TValue>> BeEquivalentTo<TExpectation>(TExpectation expectation,
```

Comment:
Maybe return `AndWhichConstraint`?

Reply:
What would the `Which` part be?

Reply:
On 2nd thought, `And` would be enough. This PR merely triggered some thoughts on #1126.